lib/DDGC/Web/Controller/My/Blog.pm

```diff
@@ -70,7 +70,7 @@ sub edit :Chained('base') :Args(1) {
 
 		my %values;
 
-		for (qw( format title uri topics teaser content live fixed_date )) {
+		for (qw( format title uri topics teaser content live fixed_date users_id )) {
```

Comment:
It's just a new field in the form if the user already has an author... I guess I don't understand the question.